### ISSUE TITLE
External Browser Problem

### ISSUE THREAD

**Issue by rafaeluriarte** [chrisbra/CheckAttach#24]
Thanks for the suggestion for using together with mutt.  However, I am having difficulties to execute another "file browser" with your plugin. Here is my configuration:

let g:checkattach_filebrowser='/usr/bin/find /home/XX/Desktop /home/XX/Documents/Dropbox   -not -path "*vim*" -type f | /usr/bin/fzf -m'

The output of the command if run it in the command line is:
/home/XX/Documents/Dropbox/EnvSimulator.class
/home/XX/Documents/Dropbox/EdgeNodeHost.class
/home/XX/Documents/Dropbox/EdgeNodeHost2.class

However, it does not even open the "file browser" and there are no error. Did I misunderstand something?

Thank you

**Comment by chrisbra** [comment]
are you trying to pipe the output back to vim (e.g. trying to run vim recursively)?

**Comment by rafaeluriarte** [comment]
No, no idea how I could do that (sorry for my ignorance).  Could it be something like?

let g:checkattach_filebrowser='fzf | vim -'

In any case, when I try to call straight :AttachFile it shows:

"unknown option: ."

**Comment by chrisbra** [comment]
It should be rather:

    let g:checkattach_filebrowser='fzf'

> In any case, when I try to call straight :AttachFile it shows:
> "unknown option: ."

I have no idea where that comes from, some strange modeline?

**Comment by nickwynja** [comment]
I just came across this issue. The "unknown option: ." is because `fzf` doesn't support a path like `.` as an argument.

I have a [working branch](https://github.com/nickwynja/CheckAttach/tree/fzf) with better `fzf` support. I need to keep testing it out but I can open a PR for it in the future.

**Comment by chrisbra** [comment]
sure, please go ahead
